test/index.js

```diff
@@ -1,5 +1,18 @@
 'use strict';
-const binding = require('./build/Release/binding.node');
-const assert = require('assert');
+global.buildType = process.config.target_defaults.default_configuration;
+global.binding = require(`./build/${buildType}/binding.node`);
+global.assert = require('assert');
```

Comment:
This means that somebody looking at the test files themselves might need to look up where those “magic” globals come from … I’d be very okay with just copying these lines into the test files themselves :)

That would also have the advantage that they are are runnable as standalone node scripts, which might be nice when the test suite is bigger and takes longer to run.